web/src/main/ui/app/components/flows-new/edit-flow/ingest/ui/ingest-ui.component.html

```diff
@@ -37,7 +37,7 @@
     ></info-label>
     <mat-form-field layout-fill appearance="outline">
       <mat-select
-        [(ngModel)]="step.options.document_type"
+        [(ngModel)]="this.step.options.documentType"
```

Comment:
'document_type should' be 'outputFileType'

Reply:
Thanks for catching that! Fixed.